activetable.xml

```diff
@@ -1,118 +1,184 @@
-<problem display_name="webGLDemo">
+<problem>
 <script type="loncapa/python">
 <![CDATA[
-def grader(e, ans):
-    state = json.loads(json.loads(ans)['answer'])
-    response = state['response']
-    colors = state['colors']
-    #return {'ok': False, 'msg': '%s' % str(response)}
-    
-    Numeric = 'NUMERIC_RESPONSE'
-    questionTable = [
-        ['Carts','v_initial','v_final','Delta v'],
-        ['green','2.0','-1.2',Numeric],
-        ['red','-2.0','1.2',Numeric],
-    ]
+# This Python code defines and evaluates the problem. You usually don't need to touch the
+# first portion of the code and can jump to "Problem definition" near the end.
 
-    answerTable = [
-        ['Carts','v_initial','v_final','Delta v'],
-        ['green','2.0','-1.2','-3.2'],
-        ['red','-2.0','1.2','3.2'],
-    ]
-    
-    ### Must remove header for comparison in loop below
-    ### JQuery takes cells without a header, so i,j values must only be fore table body.
-    answerTable = answerTable[1::]
-
-    def stringComparison(R,K):
-        '''
-        R: Response from student
-        K: Answer from (K)ey
-        '''
-        if str(R)==str(K):
-            return True
-        else:
-            return False
-    
-    def testNumeric(X):
-        '''
-        X: entry taken from Active Table
-        '''
+import json
+import re
```

Comment:
There was a version that did use it.  Alas, I can't conveniently run a linter on this code, so things like this can happen.